run-custom-mao.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -xe
+
+export SCRIPTS_DIR=$(pwd)
+export CUSTOM_MAO_BUILD=quay.io/sdasu/machine-api-operator:sdasu-fix
+
+kubectl get secrets --namespace openshift-machine-api | grep mariadb
+
+# Stop CVO, MAO, machine-api-controller and Metal3 deployments
+oc --config /home/stack/dev-scripts/ocp/auth/kubeconfig scale deployment --replicas=0 cluster-version-operator --namespace openshift-cluster-version
+oc --config /home/stack/dev-scripts/ocp/auth/kubeconfig scale deployment -n openshift-machine-api --replicas=0 machine-api-operator
+oc --config /home/stack/dev-scripts/ocp/auth/kubeconfig scale deployment --replicas=0 machine-api-controllers --namespace openshift-machine-api
+oc --config /home/stack/dev-scripts/ocp/auth/kubeconfig scale deployment --replicas=0 metal3 --namespace openshift-machine-api
+
+REPLACEMENT_TEXT="image: ${CUSTOM_MAO_BUILD}"
+echo $REPLACEMENT_TEXT
+
+oc get deployments -n openshift-machine-api machine-api-operator -o yaml > mao.yaml
+sed -i -e "s!image: .*!${REPLACEMENT_TEXT}!g" mao.yaml
```

Comment:
I see 3 images in my machine-api-operator deployment, and they aren't all the same. Is this going to replace all of them with the same image? Will that work?

```
$ oc get deployments -o yaml machine-api-controllers | grep image
        image: quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:c5c69f730625c96015d672d13ef51f08d377cfa89e27cde11c28bc4e0fdc1810
        imagePullPolicy: IfNotPresent
        image: quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:c5c69f730625c96015d672d13ef51f08d377cfa89e27cde11c28bc4e0fdc1810
        imagePullPolicy: IfNotPresent
        image: quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:e02787a081e7e14a47bdc28516ee0b9af543e5c84ec4f88aa1f1f7f89e4e2322
        imagePullPolicy: IfNotPresent
```

Reply:
We are looking at the machine-api-operator deployment and not the machine-api-controllers deployment.
[stack@dell-r630-003 dev-scripts]$ oc get deployments -n openshift-machine-api machine-api-operator -o yaml | grep "image:"
        image: quay.io/sdasu/machine-api-operator:sdasu-fix

Reply:
Oops, you're right, I copied the wrong string into my command. Sorry for the noise.